db/fns.md

```diff
@@ -93,13 +93,15 @@
    * [`get_dependent_tasks`](#get_dependent_tasks)
    * [`get_queue_artifact`](#get_queue_artifact)
    * [`get_queue_artifacts_paginated`](#get_queue_artifacts_paginated)
-   * [`get_task_group`](#get_task_group)
+   * [`get_task_group_project_ids`](#get_task_group_project_ids)
+   * [`get_task_group2`](#get_task_group2)
    * [`get_task_projid`](#get_task_projid)
    * [`get_task_queue`](#get_task_queue)
    * [`get_task_queues`](#get_task_queues)
    * [`get_tasks_by_task_group_projid`](#get_tasks_by_task_group_projid)
    * [`is_task_blocked`](#is_task_blocked)
    * [`is_task_group_active`](#is_task_group_active)
+   * [`is_task_group_sealed`](#is_task_group_sealed)
```

Comment:
What is the difference between is_task_group_active and is_task_group_sealed? I wonder if is_task_group_active was the old way to guess if a task group was still in progress, and perhaps now we can use the new method and delete the old one entirely?

Reply:
This is something different. `is_task_group_active` checks all tasks for the given `taskGroupId` and returns true if there are non-resolved tasks